forch/http_server.py

```diff
@@ -16,6 +16,14 @@
 LOGGER = logging.getLogger('httpserv')
 
 
+class HttpException(Exception):
+    """Http exception base class"""
+    def __init__(self, message, http_status):
+        Exception.__init__(self, message)
+        self.message = message
```

Comment:
self.message is redundant with the Exception base class. Also doesn't seem to be used anywhere?

Reply:
Removed